src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/UseStringContainsCharOverloadWithSingleCharactersTests.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Performance.UseStringContainsCharOverloadWithSingleCharactersAnalyzer,
+    Microsoft.NetCore.Analyzers.Performance.UseStringContainsCharOverloadWithSingleCharactersCodeFix>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Performance.UseStringContainsCharOverloadWithSingleCharactersAnalyzer,
+    Microsoft.NetCore.Analyzers.Performance.UseStringContainsCharOverloadWithSingleCharactersCodeFix>;
+
+namespace Microsoft.NetCore.Analyzers.Performance.UnitTests
+{
+    public class UseStringContainsCharOverloadWithSingleCharactersTests
+    {
+        [Fact]
+        public async Task CSharp_RegularStringLiteralWithoutCharOverloadAvailable_NoDiagnostic()
+        {
+            var sourceCode = @"using System;
+public class TestClass
+{
+    public bool TestMethod(string input)
+    {
+        return ""testString"".Contains({|#0:""a""|});
```

Comment:
I'm not sure why you have markup here given that there is no diagnostic.

Reply:
What can i say - i am lazy man. 🤣